ocaml/sdk-gen/csharp/autogen/src/Session.cs

```diff
@@ -175,6 +144,14 @@ private void SetAPIVersion()
                 Host host = Host.get_record(this, pool.master);
                 APIVersion = Helper.GetAPIVersion(host.API_version_major, host.API_version_minor);
             }
+
+            if (JsonRpcClient != null)
+            {
+                if (APIVersion == API_Version.API_2_6)
```

Comment:
Seems the default value is already "JsonRpcVersion.v1", do we need re-setting here?

Reply:
Theoretically someone could create the session object, change the JsonRpcVersion, and then call `login_with_password` which calls this method. In reality it's highly unlikely someone would do that, but I put the code there to be defensive.